src/main/webapp/assets/modules/event-details/eventDetailsController.js

```diff
@@ -1,4 +1,18 @@
-techlooper.controller("eventDetailsController", function ($scope) {
+techlooper.controller("eventDetailsController", function ($scope, $routeParams, $location, utils) {
```

Comment:
Line is too long.<br>'techlooper' is not defined.<br>'utils' is defined but never used.<br>'$location' is defined but never used.<br>'$routeParams' is defined but never used.<br>'$scope' is defined but never used.